src/configurator.rb

```diff
@@ -78,29 +84,54 @@ class Question
 
       attr_reader :identifier, :question, :type, :choices
 
-      def initialize(identifier:, properties:, configure_file:, questions_section:)
+      def initialize(identifier:, properties:, configure_file:,
+                     questions_section:, default: nil)
         @identifier = identifier
         @question = properties[:question]
         @choices = properties[:choices]
+        @default = default
         @type = type_for(
           properties[:type],
           configure_file: configure_file,
           questions_section: questions_section
         )
       end
 
+      def add_default(question, type = :string)
+        unless @default.nil?
+          parsed_default = nil
+          case type
+          when :boolean
+              if @defualt == "true" || @defualt.is_a?(TrueClass)
```

Comment:
`@defualt` != `@default`!

Also, I don't think it's the responsibility of setting defaults to try and coerce things to the correct type. It seems like extra code without a real benefit; you should specify the correct type in the `configure.yaml` file, and this will just let people write sloppy `configure.yaml`s.

I think a more correct way to handle the wrong type being used would be to add a step (earlier in `configure`) to validate the `configure.yaml` is the correct format, which would validate `default` as well as `required` are the correct type among other validations.

However since this is written it does provide some value for now, so the best step now may be to add a note to the code and a corresponding card regarding adding validation of `configure.yaml`. Thanks.